explorer/service/src/main/resources/config.properties

```diff
@@ -10,6 +10,6 @@ metrics.dump=metrics.csv
 transactions.size=50
 universe.shards=17592186044416
 universe.magic=1363935234
-test.dump=state.bak
+test.dump=service/data/state.csv
```

Comment:
We can of course say that this is a "csv" file, even though it isn't (it only contains one word, like `STARTED` or `TERMINATED` or any other valid test run state).

I realise "bak" isn't an optimal extension, but I also think "txt" or similar would be more suitable than "csv".

Reply:
One other note: Do we want to write the metrics dump to persistent storage too? If so, we need to make the corresponding adjustments to the `metrics.dump` property as well.

Reply:
A final note: The path as stated now, seems to be relative path. Is that by intention? Could we maybe add the absolute path instead to avoid confusion (`/opt/radixdlt/service/data/state.txt`)?

Reply:
And the absolutely final question: does the Explorer (user) have permissions to create and edit files in the target directory?

Reply:
good catch thanks